packages/components/src/components/Task/Task.js

```diff
@@ -52,6 +46,15 @@ class Task extends Component {
     });
   };
 
+  handleTaskSelected = event => {
+    if (event) {
+      event.preventDefault();
+    }
+    this.setState({ selectedStepId: null }, () => {
+      this.handleClick(event);
```

Comment:
```suggestion
      this.handleClick();
```

no need to pass the event since it's already been processed above, this avoids the React warning